JBoss/EAP/6/profiles/stig-eap6-upstream.xml

```diff
@@ -1,6 +1,6 @@
 <Profile id="stig-eap6-upstream">
```

Comment:
Because this is directly aligned to DISA's content, and there is no upstream/downstream relationship on the EAP content, we the profile should be renamed.... e.g. ``stig-eap6-disa`` to match the RHEL names.